src/handlers/generate-erc721-permit.ts

```diff
@@ -74,20 +75,25 @@ export async function generateErc721PermitSignature(
     _nftContractAddress = NFT_CONTRACT_ADDRESS;
     _evmNetworkId = evmNetworkId;
     _nftMinterPrivateKey = NFT_MINTER_PRIVATE_KEY;
-    const walletAddress = await adapters.supabase.wallet.getWalletByUsername(username);
+    _userId = userId;
+    const walletAddress = await adapters.supabase.wallet.getWalletByUserId(_userId);
     if (!walletAddress) {
       _logger.error("No wallet found for user");
       throw new Error("No wallet found for user");
     }
     _walletAddress = walletAddress;
-    _userId = await adapters.supabase.user.getUserIdByWallet(_walletAddress);
     if (isIssueEvent(contextOrPermitPayload)) {
       _issueId = contextOrPermitPayload.payload.issue.id.toString();
     } else {
       throw new Error("Issue Id is missing.");
     }
     _organizationName = contextOrPermitPayload.payload.repository.owner.login;
     _repositoryName = contextOrPermitPayload.payload.repository.name;
+    const { data: userData } = await contextOrPermitPayload.octokit.request("GET /user/:id", { id: _userId });
```

Comment:
I think octokit has method for fetching user instead of using paths

Reply:
It does, but only using the user name and not the user id. It we decide to change it back to the user name I will use octokit's method